addons/web/static/tests/webclient/actions/legacy_tests.js

```diff
@@ -141,6 +141,9 @@ QUnit.module("ActionManager", (hooks) => {
         assert.containsOnce(webClient, ".custom-action");
         assert.verifySteps([]);
 
+        // close debug menu
+        await click(webClient.el, ".o_debug_manager button");
+        // open debug menu
```

Comment:
is it related?

Reply:
yes, with the "true" ui service, the debug manager stays open. So we have to close it before reopening it

Reply:
this is due to the fact that the computation of activeElement was not done in the "fake" service